user/user.go

```diff
@@ -0,0 +1,60 @@
+package user
```

Comment:
shouldn't this live in `database/user.go` ?

Reply:
F/U: https://github.com/NebulousLabs/skynet-accounts/issues/5